zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1133,6 +1133,15 @@ property, when available, is noted below.
     effect due to TLS handshake timeout when there are too many in-flight TLS 
     handshakes. Set it to something like 250 is good enough to avoid herd effect.
 
+* *forward_learner_requests_to_commit_processor_disabled*
+    (Jave system property only: **zookeeper.forward_learner_requests_to_commit_processor_disabled**)
+    When this property is set, the requests from learners won't be enqueued to
+    CommitProcessor queue, which will help save the resources and GC time on 
+    leader.
+
+    The default value is false, will enable it by default after verified on prod.
```

Comment:
Two nits:
* I suspect that a config option with name "forward_learner_requests_to_commit_processor_disabled" in zoo.cfg will achieve the same effect of setting a java system property, so this is not precisely a "java system property only" (and i suspect we have a couple of other configuration options which claimed to be "java system property only" in the admin doc but they actually are not.). 

* I think we can remove the "will enable it by default after verified on prod." part.